frontend/components/PlatformCompatibility/PlatformCompatibility.tsx

```diff
@@ -22,12 +22,12 @@ const DISPLAY_ORDER = [
   "Windows",
   "Linux",
   "ChromeOS",
-] as SupportedDisplayPlatform[];
+] as QueryableDisplayPlatform[];
```

Comment:
For speed this can be on another iteration but can we still improve the typing on line 14?